### PR TITLE
genesis: enable generating validator keys

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

File: crates/sui/src/keytool.rs [hunk 1]
```diff
@@ -37,9 +42,22 @@ impl KeyToolCommand {
     pub fn execute(self, keystore: SuiKeystore) -> Result<(), anyhow::Error> {
         match self {
             KeyToolCommand::Generate => {
-                let (address, keypair) = get_key_pair();
-                store_and_print_keypair(address, keypair)
+                let (_address, keypair) = get_key_pair();
+
+                let hex = encode_bytes_hex(keypair.public_key_bytes());
+                let file_name = format!("{hex:.8}.key");
```
Comment (huitseeker) [inline crates/sui/src/keytool.rs:48]: Nit: The pubkey is 16 nibbles, which I think is an OK file name length.

**Review by huitseeker — APPROVED** [review]
LGTM

**Review by mystenmark — APPROVED** [review]

File: crates/sui/src/keytool.rs (hunk 1 above)
Comment (bmwill) [reply crates/sui/src/keytool.rs:48]: i'll change that then

**Review by kchalkias — APPROVED** [review]

**Review by Clay-Mysten — APPROVED** [review]
Markdown LGTM

**Pull request merged**